templates/1-basic_workflow_with_config/properties1.yml

```diff
@@ -1,18 +1,23 @@
+# This file bears a similar purpose to indicators1.yml, but is called during the diagnostics step.
+# Same as for indicators1.yml, this will create a custom xclim module using your own specifications.
+# For a complete list of the possible arguments, please refer to the xclim documentation: https://xclim.readthedocs.io/en/stable/api.html#indicator-tools
+
 realm: generic
 indicators:
   quantile_02-tasmin:
-    base: xclim.sdba.properties.quantile
+    base: xclim.sdba.properties.quantile  # Properties do not have an identifier, so the full path must be specified.
     cf_attrs:
-      long_name: 2th quantile of the minimum temperature
+      long_name: 2nd quantile of the minimum temperature
     input:
-      da: tasmin
+      da: tasmin  # Since properties are generic, the input name must be specified.
     parameters:
-      q: 0.98
+      q: 0.02
       group: time.season
   maximum_length_of_warm_spell:
```

Comment:
This part of the example (maximum_length_of_warm_spell) didn't work for me, but that's a different issue.

Reply:
can you make an issue to explain the problem you had, please ? then, we can try to fix it!